src/features/dataset/DatasetHeader.tsx

```diff
@@ -68,8 +70,8 @@ const DatasetHeader: React.FC<DatasetHeaderProps> = ({
             <div className='flex mt-3 text-sm'>
               <DatasetInfoItem icon='automationFilled' label='automated' iconClassName='text-qrigreen' />
               <DatasetInfoItem icon='disk' label='59 MB' />
-              <DatasetInfoItem icon='download' label='418 downloads' />
-              <DatasetInfoItem icon='follower' label='130 followers' />
+              <DatasetInfoItem icon='download' label={`${versionInfo?.downloadCount || 0} downloads`} />
+              <DatasetInfoItem icon='follower' label={`${versionInfo?.followerCount || 0} followers`} />
```

Comment:
refactoring versionInfo to always have numeric fields set makes it much easier to ship pluralization. Define label funcs at the bottom of this file:

```ts
function downloadLabel(vi?: VersionInfo) string {
  if (!vi) { return `0 downloads` }
  const verb = (vi.downloadCount === 1) ? 'download' : 'downloads'
  return `${vi.downloadCount} ${verb}`
}
```
```suggestion
              <DatasetInfoItem icon='download' label={downloadCount(versionInfo)} />
```

ditto for `followerCount`